client/devdocs/design/search-collection.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+let FilterSummary, Hider;
```

Comment:
The internal dependencies was just a carry over from copy-paste. Will remove it since there are none in this file. Will also update the let, const to be more inline with assignment as suggested.